### PR TITLE
Oc 0.6.3

### PR DESCRIPTION
**Pull request by blacklizard**

### CODE REVIEW

**Comment by blacklizard** [comment]
Oc 0.6.3

File: BIOS.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # BIOS Configuration
 
-BIOS version: `F11`
+BIOS version: `F12j`
```
Comment (gbasile) [inline BIOS.md:3]: Does this mean that now we can do the CFG Lock directly on the BIOS? 🥇

Comment (blacklizard) [reply BIOS.md:3]: @gbasile yes.. with f12j, it's been added in the BIOS GUI..

Comment (123marvin123) [reply BIOS.md:3]: Yep, apparently they added this option in F12j. I already updated to F12j and the CFG Lock option works great.

**Comment by neb0t** [comment]
I guess yes 👍 
But first - do update bios 

---

Best regards,
Maksym Nebot

**Pull request merged**